app/views/moves/_form.html.slim

```diff
@@ -8,7 +8,31 @@
     .form-group
       label.form-label From
       = from_establishment&.name
-    = f.text_field :to
+
+    .radio-concertina
+      .form-group
+        fieldset
+          legend
+            span.form-label-bold To
+
+          - Establishment::ESTABLISHMENT_TYPES.each do |option|
+            = f.radio_concertina_option(:to_type, t(option, scope: 'move.destination_types.labels'), option) do
+              label.form-label for=option
+                = t(option, scope: 'move.destination_types.prompts')
+              - if form.errors["to_#{option}"].any?
+                .error
+                  .span.error-message
+                    = form.errors["to_#{option}"].join(', ')
+                  = f.select "to_#{option}", option.to_s.classify.constantize.pluck(:name), {include_blank: true}, { id: option }
```

Comment:
option.to_s.classify.constantize.pluck(:name, :id)

with this we can still show name in the autocomplete, but store ids in the database. I would prefer this approach to store ids over strings

Reply:
After discussing on it I am happy with this approach